Formula/algernon.rb

```diff
@@ -1,8 +1,8 @@
 class Algernon < Formula
   desc "Pure Go web server with Lua, Markdown, HTTP/2 and template support"
   homepage "https://algernon.roboticoverlords.org/"
```

Comment:
The homepage isn't accessible, we could use GitHub instead:
```suggestion
  homepage "https://github.com/xyproto/algernon"
```

Reply:
The homepage does not have a valid HTTPS certificate right now, because it expired. It will be fixed very soon (hours, not days).

Reply:
awesome, no rush

Reply:
While developing, I got locked out from Let's Encrypt for a week because I unknowingly requested too many certificates.

I could try to use different certificates, but I'm tempted to just wait a week.

Using https://github.com/xyproto/algernon for now is fine, I can update the homepage URL for the next release of Algernon. :slightly_smiling_face:

Reply:
Algernon now has support for Let's Encrypt, which will be included in the next release.

https://algernon.roboticoverlords.org/ is now using that feature, so using that URL in the Homebrew package should be possible in the future. :slightly_smiling_face: